### PR TITLE
Modulo operator support

### PR DESCRIPTION
**Pull request by boki1**
Related to Issue #10 

------

# Description

Currently no modulo (`%`) operator is provided, so add that to the binary arithmetic operations supported by the compiler.

# Task

Provide implementation of the modulo (`%`) operator where required. 

# Definition of Done

A successful build of the gcd algorithm.

### CODE REVIEW

File: test/test-this-output.txt
Comment (vkabadzhova) [inline test/test-this-output.txt]: I'm not sure if these files should be commited. Maybe it is a good idea to have an example output, but I prefer to put it in the Wiki, not commit it. Please, take this into account.

File: test/test.sh
Comment (vkabadzhova) [inline test/test.sh]: Delete comments and make some clean in the Makefile

File: .gitmodules
Comment (vkabadzhova) [inline .gitmodules]: Thank you!

File: src/main.cpp
Comment (vkabadzhova) [inline src/main.cpp:8]: OK, but should be fixed in next commits. We won't merge until this is already fixed

File: test/testcases/gcd.kv
Comment (vkabadzhova) [inline test/testcases/gcd.kv:1]: Yes! :100:

File: test/testcases/fib.kv
Comment (vkabadzhova) [inline test/testcases/fib.kv:1]: Yes again! :100: :partying_face:

**Review by vkabadzhova — CHANGES_REQUESTED** [review]
First, sorry for the single comments that are reviews, it looks like I've really messed something with the reviewing order and now I only can put reviews, so just resolve them. Then good job, I like the work. I have 2 things I insist to be changed before we merge:
- [x] Remove the sample outputs in the `test/` directory and if you want samples, put them in the documentation
- [x] Please, just try to fix the visualization flag. It should not be always enabled.

File: src/main.cpp
Comment (boki1) [reply src/main.cpp:8]: This flag is set only for demonstration purposes. In future releases an argument parser would be a really good addition. So would you be OK, if this is remains unchanged for now.

Comment (vkabadzhova) [reply src/main.cpp:8]: OK

File: test/test-this-output.txt
Comment (boki1) [reply test/test-this-output.txt]: Fixed it.

File: test/test.sh
Comment (boki1) [reply test/test.sh]: Fixed it.

**Review by vkabadzhova — APPROVED** [review]

**Review by vkabadzhova — APPROVED** [review]

**Pull request merged**
